### DISCUSSION TITLE
Dashboard Lists: Add option to link to current time window, variables

### DISCUSSION THREAD

**Discussion by onefourfive** [grafana/grafana#48649] [Dashboards, Panels & Visualizations]
As a part of a larger organization with templated/repeating panels, I use the dashboard list panel to show relevant dashboards based on selected variables.

This is almost great for troubleshooting-- given selected variables, I have a handy list of related dashboards. However it gets cumbersome when I click the link and it leads. me to that dashboard's default time window, likely completely unrelated to the time window I was investigating. Also I may have to re-set some variables I set in the previous dashboard.

As a Grafana user, I'd like to click on a dashboard list link and be taken to that dashboard with the time window and variables from the previous dashboard. Dashboard list should have the same options as dashboard links.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
